cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,28 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io');
+  });
+
   it('should provide an ability to log in', () => {
+    const email = 'naxu@mailinator.com';
+    const password = 'Pa$$word123';
+
+    cy.contains('a', 'Sign in').should('exist').click();
+
+    cy.get('h1').should('contain.text', 'Sign In');
+
+    cy.contains('Sign in').click();
+
+    cy.get('h1').should('contain.text', 'Sign In');
```

Comment:
The assertion for 'Sign In' header is duplicated. It should be checked only once after navigating to the 'Sign In' page.

Reply:
> The assertion for 'Sign In' header is duplicated. It should be checked only once after navigating to the 'Sign In' page.

thanks already fixed